src/Compilers/CSharp/Test/Emit/CodeGen/IndexAndRangeTests.cs

```diff
@@ -4236,5 +4236,94 @@ static void M(Span<int> s)
             var comp = CreateCompilation(source, targetFramework: TargetFramework.Net70);
             comp.VerifyDiagnostics();
         }
+        [ConditionalFact(typeof(CoreClrOnly))]
```

Comment:
These don't need to be conditioned on `CoreClrOnly`. Just don't execute them on framework. Also, missing a newline above this.

Reply:
I've wanted to know the correct way. 
Is `#if NET` correct?

Reply:
Just use `ExecutionConditionUtil.IsCoreClr` to change whether you pass an expected output or not.

Reply:
Looks like I managed to deal with the problem. Thank you.